components/SplitScreen/SplitForm.js

```diff
@@ -199,60 +217,78 @@ function useConvertInputs(otherSymbol, forwards = true) {
-    inputValueAnj:
-      bondingPriceLoading && !convertFromAnj && editing !== 'anj'
-        ? 'Loading…'
-        : inputValueAnj,
-    inputValueOther,
+    inputValueRecipient,
+    inputValueSource,
   }
 }
 export default () => {
   const [selectedOption, setSelectedOption] = useState(0)
   const [inverted, setInverted] = useState(false)
+  const [isFinal, setIsFinal] = useState(false)
+  const [transactionHash, setTransactionHash] = useState(null)
   const forwards = useMemo(() => !inverted, [inverted])
   const openOrder = useOpenOrder()
   const claimOrder = useClaimOrder()
   const {
-    amountOther,
+    amountSource,
+    amountRecipient,
     bindOtherInput,
-    inputValueAnj,
-    inputValueOther,
+    bondingPriceLoading,
+    handleManualInputChange,
+    inputValueRecipient,
+    inputValueSource,
   } = useConvertInputs(options[selectedOption], forwards)
+  const tokenBalance = useTokenBalance(options[selectedOption])
 
   const { account } = useWeb3Connect()
   const inputDisabled = useMemo(() => !Boolean(account), [account])
-
+  const inputError = useMemo(() => Boolean(tokenBalance.lt(amountSource)))
   const converterStatus = useConverterStatus()
+
   const handleInvert = useCallback(() => {
     setInverted(v => !v)
     setSelectedOption(option => (option + 1) % 2)
   }, [])
+  const handleConvertMax = useCallback(() => {
+    handleManualInputChange(
+      formatUnits(tokenBalance, { truncateToDecimalPlace: 3 }),
+      forwards
+    )
+  }, [tokenBalance, forwards])
 
   const handleConvert = useCallback(async () => {
-    converterStatus.setStatus(CONVERTER_STATUSES.SIGNING)
-
-    // try {
-    //   const tx = await openOrder(amountOther, forwards)
-    //   converterStatus.setStatus(CONVERTER_STATUSES.PENDING)
-    //   await tx.wait()
-    //   const finalTx = await claimOrder(tx.hash, forwards)
-    //   await finalTx.wait()
-    //   converterStatus.setStatus(CONVERTER_STATUSES.SUCCESS)
-    // } catch (err) {
-    //   if (process.env.NODE_ENV === 'production') {
-    //     Sentry.captureException(err)
-    //   }
-    //   console.log(err)
-    //   converterStatus.setStatus(CONVERTER_STATUSES.ERROR)
-    // }
-    // }, [amountOther, forwards, converterStatus])
-  }, [converterStatus])
+    setIsFinal(false)
+    setTransactionHash(null)
+    try {
+      converterStatus.setStatus(CONVERTER_STATUSES.PENDING)
+      const tx = await openOrder(amountSource, forwards)
+      setTransactionHash(tx.hash)
+      converterStatus.setStatus(CONVERTER_STATUSES.SIGNING)
+      await tx.wait()
+      converterStatus.setStatus(CONVERTER_STATUSES.SUCCESS)
+      const finalTx = await claimOrder(tx.hash, forwards)
+      setIsFinal(true)
+      setTransactionHash(finalTx.hash)
+      converterStatus.setStatus(CONVERTER_STATUSES.SIGNING)
+      await finalTx.wait()
+      converterStatus.setStatus(CONVERTER_STATUSES.SUCCESS)
+    } catch (err) {
+      if (process.env.NODE_ENV === 'production') {
```

Comment:
I think this isn't necessary, if we're only configuring Sentry on production. If it's not configured with a DSN, it won't do anything.